### ISSUE TITLE
[0.13.0] DROP SERIES renders a series un-writable

### ISSUE THREAD

**Issue by samhatchett** [influxdata/influxdb#6821]
system info: influxdb official docker image / mac host
version 0.13.0 (tag:latest)

issue description: create a series with some data, `drop` the series, then write to it again. The series is empty. Expected behavior would be to see the data that was written after dropping. Dropping the series does appear to drop if from the index (`SHOW SERIES` returns empty), and writing to a series with the same name/tags appears to add it back into the index, but the points are not available.

caveat: using `DROP MEASUREMENT` does not render the series un-writable. 

steps to reproduce:

```
#!/bin/bash

echo "create db"
curl -i -XPOST 'http://localhost:8086/query' \
  -u root:root \
  --data-urlencode "q=create database test_db"
echo " "

echo "add series, add data"
curl -i -XPOST 'http://localhost:8086/write?db=test_db' \
  -u root:root \
  --data-binary 'my_metric,host=myhost value=0.64 1434055562000000000'
echo " "

echo "Select to prove there's data"
curl -G 'http://localhost:8086/query?pretty=true' \
  -u root:root  --data-urlencode "db=test_db" \
  --data-urlencode "q=select * from my_metric"
echo " "

echo "drop series"
curl -i -XPOST 'http://localhost:8086/query' \
  -u root:root  --data-urlencode "db=test_db" \
  --data-urlencode "q=drop series from my_metric where host='myhost'"
echo " "

echo "Select to prove it's empty"
curl -G 'http://localhost:8086/query?pretty=true' \
  -u root:root  --data-urlencode "db=test_db" \
  --data-urlencode "q=select * from my_metric"
echo " "

echo "add data back"
curl -i -XPOST 'http://localhost:8086/write?db=test_db' \
  -u root:root \
  --data-binary 'my_metric,host=myhost value=0.64 1434055562000000000'
echo " "

echo "Select all. Expected behavior: see some data. Actual behavior: empty."
curl -G 'http://localhost:8086/query?pretty=true' \
  -u root:root  --data-urlencode "db=test_db" \
  --data-urlencode "q=select * from my_metric"
echo " "

```

**Comment by mark-rushakoff** [comment]
The behavior you've demonstrated sounds like a bug to me. As a temporary workaround, it appears that if you change the timestamp on the second write, the point is inserted correctly.

**Comment by e-dard** [comment]
@samhatchett this appears to be fixed as of `1.0.0-beta2`, so I'm closing this out. Thanks for the report!
